astropy/utils/iers/iers.py

```diff
@@ -350,6 +350,12 @@ class because it has different requirements.
                                  'by IERS table.')
 
     def _interpolate(self, jd1, jd2, columns, source=None):
+        # Converting a zero-length Time object from UTC to UT1
+        # when an empty array is passed
+        if(isinstance(jd1, np.ndarray)):
```

Comment:
I think it would be better to do this a few lines down, adding to the if-statement already there as follows:
```
if is_scalar:
    mjd = np.array([mjd])
elif mjd.size == 0:
    # Short-cut empty input.
    return np.array([])
```